docs/articles/documentation/using-testcafe/common-concepts/concurrent-test-execution.md

```diff
@@ -21,6 +21,8 @@ These instances constitute the pool of browsers against which tests run concurre
 To enable concurrency, use the [-c or --concurrency](../command-line-interface.md#-c-n---concurrency-n)
 command line option or the [runner.concurrency](../programming-interface/runner.md#concurrency) method of the programming interface.
 
+> Important! Concurrent test execution is not supported in Microsoft Edge. This is because there is no known way to start Edge in a new window and make it open a particular URL.
```

Comment:
Did you check this [util](https://github.com/MicrosoftEdge/edge-launcher) out?

Reply:
/cc @AndreyBelym

Reply:
I have tried it, it's useless. It can't open URL in a new window. It can be used to create a "private browsing" window once, but the next URLs still will be opened in new tabs of this "private" window.

Reply:
Ok